src/native/corehost/bundle/file_entry.h

```diff
@@ -25,6 +26,7 @@ namespace bundle
     {
         int64_t offset;
         int64_t size;
+        int64_t compressedSize;
```

Comment:
We will probably have to somehow version this structure - see my other comment about 5.0 apps on 6.0 frameworks.

Reply:
Yes, having the framework-dependent scenario introduces backwards compatibility and a need to version the header.